assets/airports/eddf.json

```diff
@@ -6,940 +6,4866 @@
     },
     "icao": "EDDF",
     "iata": "FRA",
-    "magnetic_north": 1,
+    "magnetic_north": 1.8,
     "ctr_radius": 80,
     "ctr_ceiling": 13000,
     "initial_alt": 5000,
-    "position": ["N50d01m18.38", "E8d33m23.24", "365ft"],
+    "position": ["N50d01m18.38", "E008d33m23.24", "364ft"],
     "rr_radius_nm": 5.0,
-    "rr_center": ["N50d01m18.38", "E8d33m23.24"],
+    "rr_center": ["N50d01m18.38", "E008d33m23.24"],
     "has_terrain": true,
     "wind": {
-        "angle": 230,
-        "speed": 5
+        "angle": 202,
```

Comment:
Can we change this to `200`? `202` seems too precise

Reply:
lol sure, it's just an average

Reply:
I figured